system/metrics/sinks/file/fileSink.cpp

```diff
@@ -81,7 +81,7 @@ void FileMetricSink::writeMeasurementToFile(const std::shared_ptr<IMetric> &pMet
         size_t countBucketValues = values.size();
         __uint64 cumulative = 0;
 
-        for (int i=0; i < countBucketValues - 1; ++i)
+        for (size_t i=0; i < countBucketValues - 1; ++i)
```

Comment:
no mention of this type of change in commit message.

Reply:
The above line gave an error/warning with the new flags.

Reply:
Commit message should mention this